templates/cointoss/src/skills/toss.ts

```
@@ -0,0 +1,225 @@
+  },
+];
+
+export async function handleTossCreation(context: XMTPContext) {
+  const {
+    message: {
+      content: { params },
+      sender,
+    },
+  } = context;
+
+  if (params.description && params.options && !isNaN(Number(params.amount))) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)